.docs/user-guide/config.md

```diff
@@ -256,3 +256,22 @@ supported:
 Docker   | docker
 
 The volume driver `docker` is automatically activated.
+
+### VOLUME MOUNT PRE-EMPTION
+There is a capability to pre-emptively detach any existing attachments to other
+instances before attempting a mount.  This will enable use cases for
+availability where another instance must be able to take control of a volume
+without the current owner instance being involved.  The operation is considered
+equivalent to a power off of the existing instance for the device.
+
+Property Name | Environment Variable | CLI Flag
+--------------|----------------------|-------------
+`volume.mountpreempt`   | `REXRAY_VOLUME_MOUNTPREEMPT`          | `--preempt`
```

Comment:
Hi @clintonskitson,

If it's a gofig-defined flag, which it's...it's not. So if it doesn't use gofig then you shouldn't list the property name OR environment variable name as they're not defined. It appears you're *only* defining it as a flag.